bench

```diff
@@ -384,6 +386,23 @@ vicare_exec ()
 }
 
 # -----------------------------------------------------------------------------
+# Definitions specific to IronScheme
+
+ironscheme_comp ()
+{
+    # remove import statement
+    # sed -i -e 's/^(import (scheme.*)$//g' $1
+    # ${IRONSCHEME} -c $1 -o $2
```

Comment:
Sorry :( If you append the common before the implementation postlude, then I can actually precompile.